frontend/src/metabase/query_builder/selectors.js

```diff
@@ -2,6 +2,12 @@
 import { createSelector } from "reselect";
 import _ from "underscore";
 
+// Needed due to wrong dependency resolution order
+// eslint-disable-next-line no-unused-vars
+import Visualization from "metabase/visualizations/components/Visualization";
```

Comment:
You can just do `import "metabase/...";`

Reply:
True!